TEMPLATE.md

```diff
@@ -0,0 +1,81 @@
+---
+id:
+title:
+status: draft
+created:
+contributors:
+    -
```

Comment:
I'd suggest filling this out with example data:

```
---
id: 000-template
title: Swarm Template
status: draft
created: YYYY-MM-DD
contributors: 
    - @vitalik
---
```
perhaps with some kind of comment to edit this.

Reply:
I feel like this suggestion is contradicting with https://github.com/status-im/ideas/pull/175/files#r181621820 and not sure what the best way to resolve it is